transformations/numeric_to_word/requirements.txt

```diff
@@ -0,0 +1,3 @@
+phonenumbers==8.12.25
+docopt==0.6.2
+Unidecode==1.2.0
```

Comment:
This library can't be integrated. Can you please find a good substitute?

Reply:
Will try and update here asap @kaustubhdhole. Thank you for the confirmation